compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -757,6 +757,11 @@ impl<'hir> Map<'hir> {
         }
     }
 
+    pub fn get_parent_item_by_def_id(self, def_id: LocalDefId) -> OwnerId {
+        let hir = self.tcx.hir().local_def_id_to_hir_id(def_id);
+        self.get_parent_item(hir)
+    }
```

Comment:
Does this differ from `self.tcx.parent(def_id)`?

Reply:
I do not think so, I removed the method in a folloup commit

Reply:
it's still here